source/main.lua

```diff
@@ -34,10 +36,10 @@ gintScreenHeight = 768-- 1080
 
 garrCurrentScreen = {}	
 
-Lander = require "lander"
+Lander = require "objects.lander"
 Terrain = require "terrain"
 
-HUD = require "hud"
+Hud = require "hud"

```

Comment:
I know HUD goes against the coding style, but HUD is an acronym so it might make more sense to keep it as HUD rather than Hud.

Reply:
Yeah, I'm unsure what is the best way to do. Originally I named it HUD because of the very same reasoning you just gave. But in the meantime I've read through some style guides and most advise to do the first letter only thing.
Let's see what @togfoxy  thinks.

Reply:
Didn't see the thumbs up, I'll fix.